google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -12,25 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
-
 #include "google/cloud/pubsub/internal/tracing_message_batch.h"
 #include "google/cloud/pubsub/internal/message_batch.h"
 #include "google/cloud/pubsub/version.h"
 #include "google/cloud/internal/opentelemetry.h"
+#include <algorithm>
+#include <memory>
+#include <string>
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
 #include "opentelemetry/context/runtime_context.h"
 #include "opentelemetry/trace/context.h"
 #include "opentelemetry/trace/span.h"
 #include <opentelemetry/trace/semantic_conventions.h>
-#include <algorithm>
-#include <memory>
-#include <string>
+#endif  // GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
 
 namespace google {
 namespace cloud {
 namespace pubsub_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+
 void TracingMessageBatch::SaveMessage(pubsub::Message m) {
```

Comment:
We tend to group all the member-function definitions together (and in header-declaration order), and have a single, leading anonymous-namespace section.  It makes it a little easier to find things.